src/types.ts

```diff
@@ -230,6 +230,28 @@ export interface Options {
    * @default 'default'
    */
   syncMode?: 'default' | 'append' | 'overwrite'
+
+  /**
+   * Enable `globs` sort for scanned files.
+   *
+   * `unplugin-vue-components` using `tinyglobby` to scan files, which is non-deterministic.
+   */
+  sortByGlob?: true
```

Comment:
Add some warning here about performance when enabling this options, will be `O(F * M)`  where `F` is the number of files and `M` the "picomatch" search, check the `sortFilesByGlobPrecedence`  function.